src/status_im/communities/core.cljs

```diff
@@ -672,3 +675,23 @@
     {::async-storage/get {:keys keys
                           :cb   #(re-frame/dispatch
                                   [::category-states-loaded community-id hashes %])}}))
+
+(fx/defn add-role-to-member
+  {:events [:community.member/add-role]}
+  [_ community-id public-key role-id]
+  {::json-rpc/call [{:method     "wakuext_addRoleToMember"
+                     :params     [{:communityId community-id
+                                   :user        public-key
+                                   :role        role-id}]
+                     :on-success #()
+                     :on-error   #(log/error "failed to add role to member" community-id public-key role-id %)}]})
```

Comment:
This error log needs a bit of improvement. We can (and dare I say should) use data to give context in log messages. The `taoensso.timbre` library even encourages us to do that. Eventually, we should settle on some basic standards for logging messages, but for the time being, my suggestion is something along these lines:

```clojure
(log/error "Failed to add role to member"
           {:error        error
            :community-id community-id
            :public-key   public-key
            :role-id      role-id})
```